sycl/test-e2e/Matrix/XMX8/element_wise_all_sizes_no_split.cpp

```diff
@@ -0,0 +1,28 @@
+//==-------- element_wise_all_sizes_no_split.cpp  - DPC++ joint_matrix------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: matrix-xmx8
+
+// TODO: Currently fails and regularly times out on DG2. Re-enable when this has
+//       been addressed.
+// XFAIL: gpu-intel-dg2
```

Comment:
I think it should be unsupported to avoid time outs on DG2 in CI. And this should be mentioned in https://github.com/intel/llvm/issues/11912 to not forget to enable this test when the problem with time outs is addressed.
```suggestion
// UNSUPPORTED: gpu-intel-dg2
```